### ISSUE TITLE
Support configuring architecture (x86_64 or arm64) for aws_lambda_function

### ISSUE THREAD

**Issue by mlcooper** [hashicorp/terraform-provider-aws#21105]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

This request is to support selecting the architecture (`x86_64` or `arm64`) per this [picture](https://d2908q01vomqb2.cloudfront.net/da4b9237bacccdf19c0760cab7aec4a8359010b0/2021/09/06/lambda-arm-change-architecture.png) which is found in this [article](https://aws.amazon.com/blogs/aws/aws-lambda-functions-powered-by-aws-graviton2-processor-run-your-functions-on-arm-and-get-up-to-34-better-price-performance/)

### New or Affected Resource(s)

* aws_lambda_function

### Potential Terraform Configuration

```hcl
resource "aws_lambda_function" "test_lambda" {
  function_name = var.lambda_function_name
  architecture = "arm64"

  # ... other configuration ...
  depends_on = [
    aws_iam_role_policy_attachment.lambda_logs,
    aws_cloudwatch_log_group.example,
  ]
}
```

### References

* #0000

**Comment by breathingdust** [comment]
Hi @mlcooper 👋 , this is a duplicate of #21090. Support for this feature will be released later today in `v3.61.0` of the provider

**Comment by mlcooper** [comment]
Thank you! I searched for duplicate issues before I opened this but somehow didn't find that one.
